### ISSUE TITLE
Is it possible add HTTPS filtering in Blokada 

### ISSUE THREAD

**Issue by w4tadrag** [blokadaorg/blokada#130]
Hi, 

I apologize if this is the wrong place to post it.

So I was wondering if it's possible for Blokada to have HTTPS filtering? I'm getting really annoyed by these "sponsored" ads built into apps such as FB, Twitter, and Instagram. And I tried many filters no luck.

I know AdClear is able to do it by HTTPS filtering, which actually works as I noticed it was able to block Youtube app ads when I was using it. The only reason I stopped using it because it wants to track users for everything they do which is creepy as f. Thus I removed it.

**Comment by kar** [comment]
Yes it's an often requested feature, I wish to implement it, although it is a sizeable work package, and so far there are more urgent things to work on. I'm leaving this issue open. I would gladly accept any help on implementing this. :)

**Comment by linuxandria** [comment]
The lack of this is the only thing stopping me from switching from AdGuard and to blokada (I still pay for and occasionally use the blokada tunnel however :) )
